src/ducks/store.jsx

```diff
@@ -0,0 +1,18 @@
+import { legacy_createStore, applyMiddleware } from "redux";
+import localStorageMiddleware from "../middleware/localStorageMiddleware";
+import { composeWithDevTools } from "redux-devtools-extension";
```

Comment:
pls align import in some logical order
I suggest:
1st group - libraries import
2nd group - store and ducks
3rd group - components and hocs
4th group - local files